tests/integration/test_fim/test_synchronization/test_registry_responses_win32.py

```diff
@@ -187,20 +150,18 @@ def test_registry_sync_after_restart(key_name, value_name, tags_to_apply,
         ValueError: If a path or value are not in the sync event.
     """
     check_apply_test(tags_to_apply, get_configuration['tags'])
-
     key_path = os.path.join(subkey, key_name)
     value_path = os.path.join(key, key_path, value_name)
 
-    # wait until the sync is done.
-    get_sync_msgs(sync_interval)
     # stops syscheckd
     control_service('stop')
-
+    fim.create_registry(fim.registry_parser[key], subkey, fim.KEY_WOW64_64KEY)
     key_handle = fim.create_registry(fim.registry_parser[key], key_path, fim.KEY_WOW64_64KEY)
```

Comment:
This is a little confusing, I understand that the configured key must not be created prior to starting the test in order for the agent to report an empty database. Maybe we can rename `subkey` to `monitored_key`?